lsp-ui-doc.el

```diff
@@ -144,29 +145,14 @@ Only the `background' is used in this face."
   :group 'lsp-ui-doc)
 
 (defvar lsp-ui-doc-frame-parameters
-  '((left . -1)
-    (no-focus-on-map . t)
-    (min-width  . 0)
-    (width  . 0)
-    (min-height  . 0)
-    (height  . 0)
-    (internal-border-width . 1)
+  '((no-focus-on-map . t)
     (vertical-scroll-bars . nil)
     (horizontal-scroll-bars . nil)
-    (right-fringe . 0)
-    (menu-bar-lines . 0)
-    (tool-bar-lines . 0)
-    (line-spacing . 0)
-    (unsplittable . t)
-    (undecorated . t)
-    (top . -1)
-    (visibility . nil)
     (mouse-wheel-frame . nil)
-    (no-other-frame . t)
+    (no-accept-focus . nil)
```

Comment:
Why was it added explicitly?

Reply:
In our original child frame implementation, we removed the `no-accept-focus` so we can focus into child frame to scroll the document. There is little no issue with that approach though, i.e: mouse is not suddenly set focus to other frame.
However, `posframe` add `no-accept-focus` to `t` explicitly, so we need to add back `no-accept-focus . nil` here